rust/src/delta.rs

```diff
@@ -610,9 +610,8 @@ impl DeltaTable {
         self.state.min_writer_version
     }
 
-    /// Run the dry run of the Vacuum command on the Delta Table: list files no longer referenced by a Delta table and are older than the retention threshold.
-    /// We do not recommend that you set a retention interval shorter than 7 days, because old snapshots and uncommitted files can still be in use by concurrent readers or writers to the table. If vacuum cleans up active files, concurrent readers can fail or, worse, tables can be corrupted when vacuum deletes files that have not yet been committed.
-    pub fn vacuum_dry_run(&self, retention_hours: u64) -> Result<Vec<String>, DeltaTableError> {
+    /// List files no longer referenced by a Delta table and are older than the retention threshold.
+    fn gc(&self, retention_hours: u64) -> Result<Vec<&action::Remove>, DeltaTableError> {
```

Comment:
```suggestion
    fn get_stale_files(&self, retention_hours: u64) -> Result<Vec<&action::Remove>, DeltaTableError> {
```

`gc` conveys "garbage collection" to me, as in an _active_ collection of garbage like in the JVM.

I propose `get_stale_files` to match more with `get_files` and `get_files_by_partitions`

Reply:
+1